src/main/java/views/monthly_menu/MonthMenuP.java

```diff
@@ -0,0 +1,23 @@
+package views.monthly_menu;
+
+import use_cases.monthly_menu.MonthMenuOB;
+import use_cases.monthly_menu.MonthMenuOD;
+
+/**
+ * The presenter class for creating the Month Menu view. This
+ * class implements the MonthMenuOB interface. The interactor
+ * class calls this class to get the MonthMenuOD object for
+ * success and fail cases.
+ */
+public class MonthMenuP implements MonthMenuOB {
+
+    /**
+     * Pass in and return output data.
+     * @param output output to be shown in MonthMenuView
+     * @return MonthMenuOD object that contains output data
+     */
+    @Override
+    public MonthMenuOD createOutput(MonthMenuOD output) {
```

Comment:
Shouldn't the presenter be the one to create the output data object instead of receiving it and returning it immediately?

Reply:
I referenced the UserLoginCleanArchitecture repository, and in there the use case interactor creates the output data object and presenter modifies it. In my case, there's nothing in particular that I need to add to the MonthMenuOD, thus I let the presenter directly returned it.

Reply:
Ahh okok, that makes sense